qemu/tests/cfg/check_block_size.cfg

```diff
@@ -41,8 +40,14 @@
         - 4096_4096:
             no ide
             need_install = no
-            images += " stg"
-            image_boot_stg = no
+            start_vm = no
+            images = "stg"
```

Comment:
I thought that VM could not boot up with the other machine type, such as 4096_4096..pc since set` images = "stg"`.

Reply:
please check latest update